pkg/arvo/app/s3-store.hoon

```diff
@@ -89,7 +89,17 @@
   --
 ::
 ++  on-leave  on-leave:def
-++  on-peek   on-peek:def
+++  on-peek
+  ~/  %s3-peek
+  |=  =path
+  ^-  (unit (unit cage))
+  ?>  (team:title our.bowl src.bowl)
+  ?+    path  (on-watch:def path)
```

Comment:
You generally don't want to crash when scrying.

```suggestion
  ?.  (team:title our.bowl src.bowl)  ~
  ?+    path  [~ ~]
```

(`~` for "no answer", `[~ ~]` for "invalid request"/"doesn't exist")

Reply:
i'll make these changes but fwiw as far as i can tell ?> is literally everywhere in these apps.

Reply:
>everywhere in these apps

Crashing is fine (and desired) for pokes and watches, but not peeks. If you see `?>` in `+on-peek`s, please do feel free to fix those as per the above.